osm2gtfs/core/creator_factory.py

```diff
@@ -81,6 +82,20 @@ def get_stops_creator(self):
             print "Stops creator: Default"
             return StopsCreator(self.config)
 
+    def get_schedule_creator(self):
+        selector = self.selector
+        try:
+            module = importlib.import_module(
+                ".creators." + selector + ".schedule_creator_" + selector,
+                package="osm2gtfs")
+            schedule_creator_override = getattr(
+                module, "ScheduleCreator" + selector.capitalize())
+            print "Schedule creator: " + selector.capitalize()
+            return schedule_creator_override(self.config)
+        except ImportError:
+            print "Schedule creator: Default"
```

Comment:
There's two python2 prints here.

Reply:
Not only two, all of the prints in this class. Changed now.